### DISCUSSION TITLE
tuiGroup - sizing of each input element

### DISCUSSION THREAD

**Discussion by JJaloudi** [taiga-family/taiga-ui#6131] [Q&A] [answered]
Hi all, I'm looking to resize specific inputs inside of a tuiGroup. Currently I have a country code dropdown with a phone number & I'm trying to reduce the size of the dropdown so that the phone number input takes a majority of the space.

I've tried messing around with the "adaptive", "orientation" & "size" inputs but to no avail. If anyone has any advice, it'd be much appreciated!

![Screenshot 2023-11-29 at 12 05 50 PM](https://github.com/taiga-family/taiga-ui/assets/18541384/47f2cdde-28ad-40d3-b03a-e84cf832d318)

**Comment by waterplea** [comment, marked as answer]
It's a flex container. You can either set `flex: 1` for the country selection and, for example, `flex: 5` for the phone number or just define `max-width` for the country selector so the rest is taken up by the number.

**Reply by JJaloudi** [reply]
This was ultimately what I ended up doing, just wasn’t sure if you guys had a sort of “best practice” in mind. Thanks for such a quick & concise response!
